### ISSUE TITLE
Finetuning on A500

### ISSUE THREAD

**Issue by lyshuga** [octo-models/octo#18]
Hi,

In your paper, it was mentioned that you were doing finetuning with NVIDIA A500 with 24 Gb of VRAM. I was wondering what batch_size you used in that case. Thanks

**Comment by jhejna** [comment]
Hi! I ran finetuning experiments using an A5000.

I was able to use a batch size of 32, keeping the model window size at 1 (I think this is the default for the latest version). This ran at about 1-1.5 steps per second. 

For my final finetuning experiments, I used a batch size of 256 for 40K steps. To do that, you could set up gradient accumulation using optax. I'm not sure how big of an effect the batch size has on final performance. 

Hope this helps!
